plugins/modules/ilo_redfish_config.py

```diff
@@ -60,8 +66,150 @@
     description:
       - Value of the attribute to be configured.
     type: str
+  raid_details:
+    required: false
+    description:
+      - List of RAID details that need to be configured in the given server
+    type: list
+    elements: dict
```

Comment:
All new options need a `version_added` statement:
```suggestion
    elements: dict
    version_added: 6.1.0
```